server/index.js

```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const db = require('../database');
+const db = require('../database/pgdb.js');
 
 const app = express();
 const host = process.env.host || 'localhost';

```

Comment:
`process.env.HOST` is only going to work in development if you have the NPM package `dotenv'` brought into the file, right?

Reply:
Yes. Nice catch. I had added an update to this in a different branch for new relic and deployment prep. Given that I had local params, my server currently still words without the dotenv package.

Reply:
Scratch that previous comment. I'll go ahead and move dotenv into this pull instead. Thanks!